elements/lisk-p2p/src/disconnect_status_codes.ts

```diff
@@ -19,3 +19,8 @@ export const INCOMPATIBLE_PROTOCOL_VERSION_REASON =
 export const INCOMPATIBLE_PEER_CODE = 4104;
 export const INCOMPATIBLE_PEER_UNKNOWN_REASON =
 	'Peer is incompatible with the node for unknown reasons';
+
+// First case to follow HTTP status codes
+export const FORBIDDEN_CONNECTION = 4403;
+export const FORBIDDEN_CONNECTION_REASON =
+	'Peer is not allowed to request incoming connections';
```

Comment:
Can we change this to `Peer is not allowed to connect`. This will be a permanent part of the protocol so we need to be careful with the exact wording and keep it simple.